client/bzz.go

```diff
@@ -0,0 +1,86 @@
+package client
+
+import (
+	"strings"
+
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/ethersphere/swarm/log"
+	"github.com/ethersphere/swarm/storage"
+)
+
+type Bzz struct {
+	client *rpc.Client
+}
+
+// NewBzz is a constructor for a Bzz API
+func NewBzz(client *rpc.Client) *Bzz {
+	return &Bzz{
+		client: client,
+	}
+}
+
+// GetChunksBitVector returns a bit vector of presence for a given slice of chunks
+func (b *Bzz) GetChunksBitVector(addrs []storage.Address) (string, error) {
+	var hostChunks string
+	const trackChunksPageSize = 7500
+
+	for len(addrs) > 0 {
+		var pageChunks string
+		// get current page size, so that we avoid a slice out of bounds on the last page
+		pagesize := trackChunksPageSize
+		if len(addrs) < trackChunksPageSize {
+			pagesize = len(addrs)
+		}
+
+		err := b.client.Call(&pageChunks, "bzz_has", addrs[:pagesize])
+		if err != nil {
+			return "", err
+		}
+		hostChunks += pageChunks
+		addrs = addrs[pagesize:]
+	}
+
+	return hostChunks, nil
+}
+
+// GetBzzAddr returns the bzzAddr of the node
+func (b *Bzz) GetBzzAddr() (string, error) {
+	var hive string
+
+	err := b.client.Call(&hive, "bzz_hive")
+	if err != nil {
+		return "", err
+	}
+
+	// we make an ugly assumption about the output format of the hive.String() method
```

Comment:
You can also get the BzzAddr via the  `bzz_info` rpc call.  From there you want the the `BzzKey` field.